### ISSUE TITLE
Add "Error Boundary" in operations drawer

### ISSUE THREAD

**Issue by gndelia** [hemilabs/ui-monorepo#564]
When depositing, withdrawing, the drawer is shown. We could add an error boundary to it. Design [here](https://www.figma.com/design/4fVd9wneclsvYDYD95ApZ9/Hemi-Portal?node-id=5906-12149&node-type=frame&m=dev). This would contain the error to the context of the drawer (on unexpected errors).

**Comment by gabmontes** [comment]
@gndelia is this done?

**Comment by gndelia** [comment]
No. I will close it, it's a low priority and unlikely to be implemented
